drivers/src/memory_layout.rs

```diff
@@ -32,15 +32,16 @@ pub const LDEVID_TBS_ORG: u32 = 0x50003800;
 pub const FMCALIAS_TBS_ORG: u32 = 0x50003C00;
 pub const RTALIAS_TBS_ORG: u32 = 0x50004000;
 pub const PCR_LOG_ORG: u32 = 0x50004400;
-pub const FUSE_LOG_ORG: u32 = 0x50004800;
-pub const BOOT_STATUS_ORG: u32 = 0x50004BE4;
-pub const CFI_VAL_ORG: u32 = 0x50004BE8;
-pub const CFI_MASK_ORG: u32 = 0x50004BEC;
-pub const CFI_XO_S0_ORG: u32 = 0x50004BF0;
-pub const CFI_XO_S1_ORG: u32 = 0x50004BF4;
-pub const CFI_XO_S2_ORG: u32 = 0x50004BF8;
-pub const CFI_XO_S3_ORG: u32 = 0x50004BFC;
-pub const DATA_ORG: u32 = 0x50004C00;
+pub const MEASUREMENT_LOG_ORG: u32 = 0x50004800;
+pub const FUSE_LOG_ORG: u32 = 0x50004C00;
+pub const BOOT_STATUS_ORG: u32 = 0x50005000;
```

Comment:
(note for a future PR)

The runtime still needs to store DPE state in PersistentData. We should consider moving these non-persistent values far away (ideally before) the PersistentData struct.

Because these values are accessed out-of-band by unsafe code, they cannot be "in the middle" of PersistentData without violating the LLVM memory model. If we don't move them we'll be forced to split PersistentData into two separate structs.

Reply:
Will resolve in a future PR.